src/Translator/assets/test/formatters/formatter.test.ts

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
For this file, I've reused the same tests data from https://github.com/symfony/symfony/blob/015d5015e353ee5448bf7c350de0db4a03f8e13a/src/Symfony/Contracts/Translation/Test/TranslatorTest.php